src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs

```diff
@@ -346,7 +346,7 @@ internal static string FormatLiteral(sbyte value, ObjectDisplayOptions options)
             }
             else
             {
-                return value.ToString(CultureInfo.InvariantCulture);
+                return value.ToString(FormatCulture(options));
```

Comment:
Consider something like "GetFormatCulture".  This way it sounds like it's the culture that's being formatted.